spidermon/contrib/scrapy/monitors.py

```diff
@@ -6,6 +6,7 @@
 SPIDERMON_MAX_ERRORS = "SPIDERMON_MAX_ERRORS"
 SPIDERMON_EXPECTED_FINISH_REASONS = "SPIDERMON_EXPECTED_FINISH_REASONS"
 SPIDERMON_UNWANTED_HTTP_CODES = "SPIDERMON_UNWANTED_HTTP_CODES"
+SPIDERMON_UNWANTED_HTTP_CODES_THRESHOLD = "SPIDERMON_UNWANTED_HTTP_CODES_THRESHOLD"
```

Comment:
Don't knoe if threshold is good name, maybe `SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT`.